app/helpers/dcv/media_element_helper.rb

```diff
@@ -1,7 +1,7 @@
 module Dcv::MediaElementHelper
   # render a video streaming player with a non-token protected src
   def render_media_element_streaming_video_player(wowza_project, video_path, poster_path, captions_path: nil, width:1024, height:576, logo_path: nil, **args)
-    url = "https://firehose.cul.columbia.edu:8443/#{wowza_project}/_definst_/mp4:#{video_path}/playlist.m3u8"
+    url = "https://diglib-wowza-prod1.cul.columbia.edu:8443/#{wowza_project}/_definst_/mp4:#{video_path}/playlist.m3u8"
```

Comment:
It's good to update this but I don't think there's a reason to hard-code the wowza hostname.

Reply:
Just accepted your GitHub suggested change!  Thanks!